### PR TITLE
Add Caldor Fire dataset

### PR DESCRIPTION
**Pull request by ividito**
Addresses https://github.com/NASA-IMPACT/veda-data-pipelines/issues/216

data-pipelines PR: https://github.com/NASA-IMPACT/veda-data-pipelines/pull/239

### CODE REVIEW

File: datasets/caldor-fire-characteristics-burn-severity.data.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        - "#781D6D"
+        - "#34095F"
+  
+---
+
+<Block>
+<Prose>
+  ## About
+
+  `.geojson` and `tif` files describing the progression and active fire behavior of the 2021 Caldor Fire in California via the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. This includes an extra `.tif` file detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.
```
Comment (j08lue) [inline datasets/caldor-fire-characteristics-burn-severity.data.mdx:67]:
```suggestion
This dataset describes the progression and active fire behavior of the 2021 Caldor Fire in California, analyzed via the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. It includes an extra layer detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.
```

**Review by j08lue — APPROVED** [review]
Looks good to me, with a little adjustment to the "About" text, so it does not mention tif og geojson files.

**Comment by j08lue** [comment]
We need a subject matter expert to review that things look right - @ashiklom maybe?:

image

**Comment by ashiklom** [comment]
I'll take a look. Also, tagging in @eorland @scoffiel @mccabete to weigh in.

File: datasets/caldor-fire-characteristics-burn-severity.data.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        - "#781D6D"
+        - "#34095F"
+  
+---
+
+<Block>
+<Prose>
+  ## About
+
+  This dataset describes the progression and active fire behavior of the 2021 Caldor Fire in California, analyzed via the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. It includes an extra layer detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.
```
Comment (ashiklom) [inline datasets/caldor-fire-characteristics-burn-severity.data.mdx:67]: @eorland Take a look at this text.

Comment (eorland) [reply datasets/caldor-fire-characteristics-burn-severity.data.mdx:67]: Text looks good! I would _maybe_ change the words "analyzed via" to the phrase "as recorded by" but this really is me splitting hairs.

**Comment by eorland** [comment]
OK - so I think I have it figured out. 

If you convert it from the native `EPSG:26910` projection to `WGS84`, some small masking needs to occur: 

```
frp = rio.open_rasterio('/projects/my-public-bucket/caldor_veda_viz_files/frp_to_save.tif') # Alexey's public bucket
bs = rio.open_rasterio('/projects/my-public-bucket/caldor_veda_viz_files/bs_to_save.tif')
frp = frp.rio.reproject('EPSG:4326')
bs = bs.rio.reproject('EPSG:4326')
```
`bs.where(bs<5).plot()` # BS values are 1-4. 255 needs to be masked out

bs_mask_Ex

`frp.where(frp>0).plot()` # FRP values are numbers >0. 

frp_mask_ss

When I originally generated these files, the values that need to be masked out here were not present, but seem to be added during reprojection. I hope this helps and I can try to supply new files in `WGS84` with the proper masks if needed!

**Comment by slesaad** [comment]
@eorland 
> I can try to supply new files in WGS84 with the proper masks if needed!

It would be great if you could! Thank you so much! 🙇

**Comment by eorland** [comment]
> It would be great if you could! Thank you so much! 🙇

No problem! They're available at: 
`/projects/shared-buckets/ashiklom/caldor_veda_viz_files/frp_to_save_epsg4326.tif`

and

`/projects/shared-buckets/ashiklom/caldor_veda_viz_files/bs_to_save_epsg4326.tif`

Sorry for the unforseen challenges and please let me know if there are any issues.

**Comment by slesaad** [comment]
@eorland could you replace the files in the veda store with the new ones? I'm not sure if I know how to access the directories you shared? Thanks!

**Comment by eorland** [comment]
@slesaad Moved!

File: datasets/caldor-fire-characteristics-burn-severity.data.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    description: ""
+    initialDatetime: newest
+    zoomExtent:
+      - 10
+      - 20
+    sourceParams:
+      colormap_name: inferno_r
+      rescale:
+        - -41
+        - 98
```
Comment (slesaad) [inline datasets/caldor-fire-characteristics-burn-severity.data.mdx:27]: The new files apparently have a different min max value? (ref: the image in this PR conversation).
I reran the ingestion pipeline - so the min max value have changed:
check here: https://staging-stac.delta-backend.com/collections/caldor-fire-behavior/items

Comment (slesaad) [inline datasets/caldor-fire-characteristics-burn-severity.data.mdx:27]:
```suggestion
        - 0
        - 93
```

File: datasets/caldor-fire-characteristics-burn-severity.data.mdx (hunk 2 above)
Comment (slesaad) [inline datasets/caldor-fire-characteristics-burn-severity.data.mdx:67]:
```suggestion
  This dataset describes the progression and active fire behavior of the 2021 Caldor Fire in California, as recorded by the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. It includes an extra layer detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.
```

File: datasets/caldor-fire-characteristics-burn-severity.data.mdx (hunk 3 above)
Comment (eorland) [reply datasets/caldor-fire-characteristics-burn-severity.data.mdx]: @slesaad thank you for making all these changes! I have one other note related to the data scaling: the burn severity raster only has valid values from 1-4. Is it possible to change those values accordingly? I also saw your note on whether `inferno` or `inferno_r` would be better... honestly I think either work, but your comment about warmer colors indicating higher severity values makes sense to me. 

Last thing: is it possible to replace "Fire Behavior" with "Mean Fire Radiative Power (MW)"?

**Pull request merged**